sessions/stats-analyses-basic.qmd

```diff
@@ -98,30 +103,63 @@ analyses.
 
 <!-- TODO: Create a revealjs presentation on this? -->
 
-:::: callout-note
+:::::::: callout-note
```

Comment:
```suggestion
:::: callout-note
```

I think this was automatically added because of the multiple callouts below, so might not be auto added again.